### PR TITLE
Fix #9474 adjusted geo processing locales

### LINKED ISSUE #9474: Geo processing tool - Advanced Settings options issue
## Description

In Advanced Settings, the option names are incorrect and are not translated correctly.

https://github.com/geosolutions-it/MapStore2/assets/56537133/2adb043a-5040-4801-bd56-132634af3767

## Other useful information

Related to https://github.com/geosolutions-it/MapStore2/pull/9351#issuecomment-1731528534

### PR DESCRIPTION
**Pull request by MV88**
## Description

Fixed locales

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

Fix #9474

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/translations/data.en-US.json [hunk 1]
```diff
@@ -3904,18 +3904,18 @@
...
           "intersectionFeature": "Intersection Feature",
           "advancedSettings": "Advanced Settings",
-          "quadrantSegments": "quadrantSegments",
-          "quadrantSegmentsTooltip": "Quadrant Segments",
-          "capStyle": "capStyle",
-          "capStyleTooltip": "Style for the buffer end caps",
+          "quadrantSegments": "Quadrant segments",
+          "quadrantSegmentsTooltip": "Number of segments",
+          "capStyle": "Cap style",
+          "capStyleTooltip": "Stil für die Pufferendkappen",
```
Comment (tdipisa) [inline web/client/translations/data.en-US.json:3910]: wrong language

File: web/client/translations/data.it-IT.json [hunk 2]
```diff
@@ -3893,18 +3893,18 @@
...
           "capStyleTooltip": "Style da usare",
           "round": "Rotondo",
           "flat": "Piatto",
           "square": "Squadrato",
-          "firstAttributeToRetain": "firstAttributeToRetain",
-          "secondAttributeToRetain": "secondAttributeToRetain",
-          "intersectionMode": "intersectionMode",
-          "percentagesEnabled": "percentagesEnabled",
-          "areasEnabled": "areasEnabled",
+          "firstAttributeToRetain": "Primo attributo da conservare",
```
Comment (tdipisa) [inline web/client/translations/data.it-IT.json:3903]:
```suggestion
          "firstAttributeToRetain": "Primo attributo da includere",
```

File: web/client/translations/data.it-IT.json [hunk 3]
```diff
@@ -3893,18 +3893,18 @@
...
+          "secondAttributeToRetain": "Secondo attributo da conservare",
```
Comment (tdipisa) [inline web/client/translations/data.it-IT.json:3904]:
```suggestion
          "secondAttributeToRetain": "Secondo attributo da includere",
```

File: web/client/translations/data.fr-FR.json [hunk 4]
```diff
@@ -3893,18 +3893,18 @@
...
           "round": "Ronde",
           "flat": "Plate",
           "square": "Carré",
-          "firstAttributeToRetain": "firstAttributeToRetain",
-          "secondAttributeToRetain": "secondAttributeToRetain",
-          "intersectionMode": "intersectionMode",
-          "percentagesEnabled": "percentagesEnabled",
-          "areasEnabled": "areasEnabled",
+          "firstAttributeToRetain": "Premier attribut à conserver",
+          "secondAttributeToRetain": "Deuxième attribut à retenir",
```
Comment (tdipisa) [inline web/client/translations/data.fr-FR.json:3904]:
```suggestion
          "secondAttributeToRetain": "Deuxième attribut à inclure",
```

File: web/client/translations/data.fr-FR.json [hunk 5]
```diff
@@ -3893,18 +3893,18 @@
...
           "capStyleTooltip": "Style pour les embouts de tampon",
           "round": "Ronde",
           "flat": "Plate",
           "square": "Carré",
-          "firstAttributeToRetain": "firstAttributeToRetain",
-          "secondAttributeToRetain": "secondAttributeToRetain",
-          "intersectionMode": "intersectionMode",
-          "percentagesEnabled": "percentagesEnabled",
-          "areasEnabled": "areasEnabled",
+          "firstAttributeToRetain": "Premier attribut à conserver",
```
Comment (tdipisa) [inline web/client/translations/data.fr-FR.json:3903]:
```suggestion
          "firstAttributeToRetain": "Premier attribut à inclure",
```

**Review by tdipisa — CHANGES_REQUESTED** [review]

File: web/client/translations/data.en-US.json (hunk 1 above)
Comment (tdipisa) [inline web/client/translations/data.en-US.json:3910]:
```suggestion
          "capStyleTooltip": "Style for the buffer end caps",
```

**Review by tdipisa — CHANGES_REQUESTED** [review]

**Review by tdipisa — APPROVED** [review]

**Comment by tdipisa** [comment]
@MV88 please backport to 2023.02.xx. Thank you.

**Comment by ElenaGallo** [comment]
Test passed, @MV88 please backport to 2023.02.xx. Thank you.

**Pull request merged**